pkg/resources/marketplace/manifestHelper.go

```diff
@@ -270,17 +269,28 @@ func ReadAndFormatManifestYamlFile(path string) (string, error) {
 
 // Gets the version number from the package yaml string
```

Comment:
Could we add some detail to the comment here about the new version syntax we allowing for.